### ISSUE TITLE
Screen reader fails to provide search results information.

### ISSUE THREAD

**Issue by msftedad** [missive/emoji-mart#714]
**Description**
Screen reader fails to provide search results information.
**Steps to reproduce:** 

1. Open URL: [https://ui.ads-int.microsoft.com/Campaign/mca/ads/campaign/14362371719539?isDMC=false&uid=194363245&cid=313974029&aid=391890532](url) in edge browser.
2. Sign in using valid credentials.
3. Navigate to Social tab and activate it.
4. Navigate to the left navigation pane and activate inbox control.
5. Make sure 'To review' tab is selected. Navigate to New post control and activate it.
6. Navigate on create new post screen and reach at the controls in Text section.
7. Turn on NVDA.
8. Navigate and activate 'Emoji icon in text field where popup opens.
9 Navigate to search field to enter data where search results appear.
10. Observe an issue that screen reader fails to provide search results information.
**Actual behavior:**  

Screen reader fails to provide search results information.

**Expected behavior:**  
Content which is visually displayed must also be available to assistive technology users. If the content is dynamic (such as an error message), focus management and/or aria-live can expose it to a screen reader. If the content is static, be sure it is not hidden with CSS or using aria hidden.

**Device & Version number**: OS: Windows_11 
Version: 22H2
OS Build: 22610.1

**Assistive device/app version:** NVDA (2021.3)

**Program/App version:** Microsoft Edge (103.0.1264.71)

**Notes:** 

NA

https://user-images.githubusercontent.com/93735775/195995262-a4c2c68a-e9bc-4d44-bd2c-e9841aa8cf3b.mp4
